.github/workflows/security.yaml

```diff
@@ -0,0 +1,34 @@
+name: security
+
+on:
+  schedule:
+    - cron: '30 1 * * 0'
```

Comment:
we probably don't need to run it as often as once every half hour, that's pretty frequent. Maybe once a day at minimum?

Reply:
yet, now it is  runs every sunday at 1:30 UTC. i will change to scan every day :).
the result if you integrate with every pr, it will on each pr checks button.
now we skip scan every pr, and the result is in GitHub Security tab.
i cope the descheduler code to my repository, and you can see result here: https://github.com/JaneLiuL/security-test-descheduler/security/code-scanning


but i still suggest that we need to **integrate with each pr and each merge to main or master and release branch**. we can not fail the pr or the pipeliens , because we can easily to check which pr import a new critical or high security issue and not fail the pr.

Reply:
sure, update done now :)